pkg/storaged/dialog.jsx

```diff
@@ -1106,7 +1106,8 @@ export const BlockingMessage = (usage) => {
         pvol: _("physical volume of LVM2 volume group"),
         "mdraid-member": _("member of MDRAID device"),
         vdo: _("backing device for VDO device"),
-        "stratis-pool-member": _("member of Stratis pool")
+        "stratis-pool-member": _("member of Stratis pool"),
+        "btrfs-member": _("member of btrfs volume")
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231208-171735-1465bfd6-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)